exporter/loadbalancingexporter/README.md

```diff
@@ -63,6 +63,12 @@ Refer to [config.yaml](./testdata/config.yaml) for detailed examples on using th
 * The `k8s` node accepts the following optional properties:
   * `service` Kubernetes service to resolve, e.g. `lb-svc.lb-ns`. If no namespace is specified, an attempt will be made to infer the namespace for this collector, and if this fails it will fall back to the `default` namespace.
   * `ports` port to be used for exporting the traces to the addresses resolved from `service`. If `ports` is not specified, the default port 4317 is used. When multiple ports are specified, two backends are added to the load balancer as if they were at different pods.
+* The `awsCloudMap` node accepts the following properties:
```

Comment:
I know nothing about AWS Cloud Map, is there anything users have to be aware before configuring it?

Reply:
The only relevant topic I can think of is that usage of the SDK hits the API every polling interval, so users have to take into account the AWS API Rate limits in their configuration.

Reply:
Can this comment be added to the readme? Or is it so obvious that it doesn't have to?